scripts/createIndexFile.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * There are times in Atlantis where we need to create our entry points
+ * programatically. One example of this is the `@jobber/hooks` package.
+ * In the hooks package, we programatically want to create the entry point
+ * to prevent contributors forgetting to export their newly created hooks.
+ */
+
+/* eslint-env node */
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require("fs");
+const path = require("path");
+
+const params = process.argv.slice(2);
+
+if (params.length > 1) {
+  throw Error(
+    "Enter only the direcory that you want to create an entry point for.",
+  );
+}
```

Comment:
Should there be a check here if the length of params is 0?  It gives a type error later on otherwise.